sp-sim/src/config.rs

```diff
@@ -0,0 +1,76 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+/*!
+ * Interfaces for parsing configuration files and working with a simulated SP
+ * configuration
+ */
```

Comment:
is there a reason you switched to C-style comments in this file?

Reply:
Oof. All the C-style comments are in chunks of code I copy-paste-modified from elsewhere in omicron; all the C++-style comments are ones I wrote by hand. I was about to write "I should've stuck with the existing style", but after some grepping it looks like I'm not the only one [mixing and matching](https://github.com/oxidecomputer/omicron/blob/dff2af73df12d907f84ea3e85559d92ab0cf9324/nexus/src/external_api/views.rs#L79-L90). This isn't something I particularly care about (except that I have a habit of using `///`) - should there be style guidance more generally around this?